includes/Classifai/Providers/Azure/ComputerVision.php

```diff
@@ -528,6 +528,14 @@ protected function scan_image( $image_url, array $routes = [] ) {
 		$settings = $this->get_settings();
 		$url      = $this->prep_api_url( $routes );
 
+		/*
+		 * MS Comuter Vision requires full image URL. So, if the file URL is relative,
+		 * then we transform it into a full URL.
```

Comment:
```suggestion
		 * MS Computer Vision requires full image URL. So, if the file URL is relative,
		 * then we transform it into a full URL.
```

Reply:
Resolved in [51bd868](https://github.com/10up/classifai/pull/350/commits/51bd8688a2fdeec4b6133a522a2bd48fca5b72a9)